lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2522,6 +2522,18 @@ namespace {
           VarDecl *pointeeProperty =
               synthesizer.makeDereferencedPointeeProperty(
                   getterAndSetter.first, getterAndSetter.second);
+
+          // Import the attributes from dereference operator clang decl to
+          // pointee property swift decl.
+          FuncDecl *setterImpl = getterAndSetter.second;
+          FuncDecl *getterImpl = getterAndSetter.first ? getterAndSetter.first
```

Comment:
IINM, this code is retrievign either getter or setter decl right? If so, I think `getterOrSetterImpl` would be a better name for `getterImpl`.

Reply:
Yes, I agree. If setter (aka getterAndSetter.second) is not present then we are using getter. For dereference operators I think setter as also a getter. But getter is just a getter. That's why this logic. 
I should rename the variable to getterOrSetterImpl.